ingestion-beam/src/main/java/com/mozilla/telemetry/schemas/SchemaAlias.java

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package com.mozilla.telemetry.schemas;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+abstract class SchemaAlias {
+
+  @JsonCreator
+  static SchemaAlias create(@JsonProperty(value = "base", required = true) String base,
+      @JsonProperty(value = "alias", required = true) String alias) {
```

Comment:
The verbosity needed for the jackson annotations is annoying and kind of ruins the brevity benefit of AutoValue classes, but I still thank you for making this change as AutoValue offers nice protection from some of the subtle bugs that can creep in when managing fields by hand.

Reply:
Still, that's a little bit less boilerplate.